modules/puppet_proxy/puppet_api.rb

```diff
@@ -70,4 +70,22 @@ def puppet_setup(opts = {})
       log_halt 406, "Failed to show puppet classes: #{e}"
     end
   end
+
+  get "/class_count" do
```

Comment:
Can we keep just the "/environments/class_count" end point? You could include a "total" key in it...